src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetEventTriggerOperation.java

```diff
@@ -108,11 +104,11 @@ public CaseEventTrigger executeForDraft(String uid, String jurisdictionId, Strin
         verifyRequiredAccessExistsForCaseType(eventTriggerId, caseType, userRoles);
 
         return filterCaseFieldsByCreateAccess(caseType, userRoles, getEventTriggerOperation.executeForDraft(uid,
-            jurisdictionId,
-            caseTypeId,
-            draftReference,
-            eventTriggerId,
-            ignoreWarning));
+                                                                                                            jurisdictionId,
```

Comment:
same as above

Reply:
Same as above 😄 
It will likely be changed when the story to get event for case is brought up.